osu.Game/Screens/OnlinePlay/Multiplayer/MultiplayerMatchSubScreen.cs

```diff
@@ -274,5 +352,10 @@ protected override void Dispose(bool isDisposing)
             if (client != null)
                 client.LoadRequested -= onLoadRequested;
         }
+
+        private class UserModSelectOverlay : ModSelectOverlay
+        {
+            protected override bool AllowCustomisation => false;
```

Comment:
Is this correct? Does this mean that free mods cannot be customised by anyone (neither the host or each individual player)? I imagine that turns mods like difficulty adjust quite useless.

Reply:
Again, this is kinda temporary because it's hard to work in. From the guest's perspective, it requires somehow clamping mod settings based on what the host picks, which I don't know if it's even possible with the current structure of things.

From the host's perspective, it requires handling of mod settings to propagate to the server, and a bit more work on `APIMod`'s equality comparer. I think this is better left to a further PR.